tests/shared/models/flash-state.spec.js

```diff
@@ -378,7 +411,8 @@ describe('Model: flashState', function () {
 
         m.chai.expect(flashState.getFlashState()).to.not.deep.equal({
           percentage: 0,
-          speed: 0
+          speed: 0,
+          humanSpeed: '0 B/s'
```

Comment:
Apologies if this is a stupid question, but why do some unit-tests have:
```
          speed: 0,
          humanSpeed: ''
```
and other unit-tests have:
```
          speed: 0,
          humanSpeed: '0 B/s'
```
?